addons/udes_stock/models/stock_picking.py

```diff
@@ -2766,3 +2766,10 @@ def _get_pick_details(self):
             "submitted_value": self.origin,
         }
         return pick_details
+
+    def unassign_users_from_pickings(self, skip_users=None):
```

Comment:
The usage of this method isn't very clear to me - is it the case that it will unassign _all_ users from the picking(s) (that is, `self`), unless they are in `skip_users`?

Reply:
Yeah, so basically, i just reused this method in two places. Once where i wanted to remove everyone from inside the pickings, and eveyone ecept those in skip_users. Changed to kwargs now anyway.

Reply:
We are already in a picking object, do we need from picking?

Reply:
Can we explain how it should be called in the docstring?  Replacing `skip_users` with `kwargs` doesn't help the reader understand what arguments the method expects, nor how they will be used.

Reply:
Well it does nothing and shouldn't' be called unless udes_kafka_events is installed which i don't mention as that is not in open. Turns out no longer need it as the from tend calls unassign user from the user models. 
But this is the case for the methods with kwargs in res_users - make the frontend happy but to do nothing without udes_kafka-events